applications/GeoMechanicsApplication/python_scripts/geomechanics_Pw_solver.py

```diff
@@ -0,0 +1,191 @@
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Import applications
+import KratosMultiphysics.GeoMechanicsApplication as KratosGeo
+import KratosMultiphysics.StructuralMechanicsApplication as KratosStructure
+
+# Import base class file
+from KratosMultiphysics.GeoMechanicsApplication.geomechanics_U_Pw_solver import UPwSolver
+
+def CreateSolver(model, custom_settings):
+    return PwSolver(model, custom_settings)
+
+class PwSolver(UPwSolver):
+    '''Solver for the solution of displacement-pore pressure coupled problems.'''
+
+    def __init__(self, model, custom_settings):
+        super(PwSolver,self).__init__(model, custom_settings)
```

Comment:
```suggestion
        super().__init__(model, custom_settings)
```

was only needed for python2 which we no longer support

Reply:
done